python/kserve/kserve/storage/storage.py

```diff
@@ -117,7 +117,7 @@ def _update_with_storage_spec():
                 ("AWS_ACCESS_KEY_ID", "access_key_id"),
                 ("AWS_SECRET_ACCESS_KEY", "secret_access_key"),
                 ("AWS_DEFAULT_REGION", "region"),
-                ("AWS_CA_BUNDLE", "certificate"),
```

Comment:
why not just call it `ca_bundle`, following the same pattern as other fields and [aws](https://docs.aws.amazon.com/sdkref/latest/guide/feature-gen-config.html#:~:text=ca_bundle%20%2D%20shared%20AWS%20config%20file%20setting)?

Reply:
that's a good idea. I will update it.